other/upgrade.php

```diff
@@ -3755,7 +3756,7 @@ function template_upgrade_below()
 	</div><!-- #footerfix -->
 	<div id="footer">
 		<ul>
-			<li class="copyright"><a href="https://www.simplemachines.org/" title="Simple Machines Forum" target="_blank" rel="noopener">SMF &copy; 2018, Simple Machines</a></li>
+			<li class="copyright"><a href="https://www.simplemachines.org/" title="Simple Machines Forum" target="_blank" rel="noopener">SMF &copy; ' . $GLOBALS['software_year'] . ', Simple Machines</a></li>
```

Comment:
Without testing i would guess,
that $GLOBALS is not defined here or?

Reply:
$GLOBALS is a [superglobal](http://php.net/manual/en/language.variables.superglobals.php), so it is always already defined and available everywhere.

However, I'm about to push another commit that switches to using a constant for this, because this value is never supposed to be changed during the execution of the script.